pkg/disk/v1alpha1/kubernetes_client.go

```diff
@@ -18,18 +18,18 @@ package v1alpha1
 
 import (
 	apis "github.com/openebs/maya/pkg/apis/openebs.io/ndm/v1alpha1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
What if we do not change this file ? Are we using this `pkg/disk/v1alpha1` elsewhere for block device requirement?

Reply:
Same as above